src/services/BalanceUtilities.tsx

```diff
@@ -130,16 +139,16 @@ class BalanceUtilities {
     )
   }
 
-  public static asKiltCoin(balance: BN): number {
-    return balance.div(new BN(KILT_FEMTO_COIN)).toNumber()
+  public static asKiltCoin(balance: BN): BN {
+    return balance
```

Comment:
Didn't we want to keep the division here?

Reply:
If we keep the division, we don't have to specify the `15` here: https://github.com/KILTprotocol/demo-client/pull/284/files#diff-58632b6ab5157d91c36e35cbc5ba2b8cR37

Reply:
We have to remove it. If we divide the value, it loses the previous integers. 

To be honest, it doesn't do anything now, I could completely remove it. Since everything filters to `KiltToken.tsx` so it seems better to keep the balance raw and handle just the way we view it.